diff.py

```diff
@@ -1813,6 +1813,7 @@ class ArchSettings:
         r"(\b|-)([0-9]+|0x[0-9a-fA-F]+)\b(?!\(r1)|[^ \t,]+@(l|ha|h|sda21)"
     ),
     re_reloc=re.compile(r"R_PPC_"),
+    arch_flags=["-m", "powerpc", "-M", "gekko"],
```

Comment:
```suggestion
    arch_flags=["-m", "powerpc", "-M", "broadway"],
```
to match permuter; I think behavior should be identical